closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -1097,8 +1097,8 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_DestroyContext
  */
 JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_CBCInit
   (JNIEnv *env, jclass thisObj, jlong c, jint mode, jbyteArray iv, jint iv_len,
-  jbyteArray key, jint key_len)
-{
+  jbyteArray key, jint key_len, jboolean doReset) {
+
```

Comment:
Please revert this, leaving the opening brace on a line by itself.

Reply:
Reverted the location change for the `{` in latest commit.